app/controllers/api_docs/services_controller.rb

```diff
@@ -124,4 +123,17 @@ def show
 
     render json: api_file
   end
+
+  private
+
+  def allowed_api?(api)
```

Comment:
Note for later.
This `#apis` method should be a service. Passing the current account and the current user.
And it should return the list of API one can see depending on its permission.
Related to this discussion: https://github.com/3scale/porta/pull/593/files#r260594144